src/portfft/dispatcher/subgroup_dispatcher.hpp

```diff
@@ -117,8 +116,8 @@ PORTFFT_INLINE void subgroup_impl(const T* input, T* output, const T* input_imag
   T wi_private_scratch[detail::SpecConstWIScratchSize];
   T priv[detail::SpecConstNumRealsPerFFT];
 #else
-  T wi_private_scratch[2 * wi_temps(detail::MaxComplexPerWI)];
-  T priv[2 * MaxComplexPerWI];
+  T wi_private_scratch[2 * wi_temps(detail::MaxComplexPerWI)]{};
+  T priv[2 * MaxComplexPerWI]{};
```

Comment:
Just add a comment as to why it is zero initialized

Reply:
addressed in https://github.com/codeplaysoftware/portFFT/pull/142/commits/48ed41fa16b8b16eb53efc3d37fcf8be5592968b